schema.sql

```diff
@@ -13,3 +13,30 @@ ALTER TABLE
 ADD
     COLUMN species VARCHAR(100);
 
+CREATE TABLE IF NOT EXISTs owners(
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(60),
+    age INT
+);
+
+CREATE TABLE IF NOT EXISTS species(id SERIAL PRIMARY KEY, name VARCHAR(50));
+
+ALTER TABLE
+    animals
+ADD
+    species_id INT;
+
+ALTER TABLE
+    animals
+ADD
+    owner_id INT;
```

Comment:
- [ ] Kindly also make sure you have the query on how you have deleted the column `species` in the `animals` table because I can only see you have how to add the `species_id` and `owner_id`